code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -45,10 +44,10 @@
 			M.gib()
 
 
-/obj/machinery/gibber/New()
-	..()
+/obj/machinery/gibber/Initialize()
+	. = ..()
 	add_overlay("grjam")
-	var/obj/item/weapon/circuitboard/machine/B = new /obj/item/weapon/circuitboard/machine/gibber(null)
+	var/obj/item/weapon/circuitboard/machine/gibber/B = new
```

Comment:
I believe they go inside with the next line where they are actually applied(I might be wrong though).